grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -186,8 +198,8 @@ public static void deleteKeyspace(String keyspace){
      * only consists of types, the inserts are idempotent and it is safe to load it
      * multiple times.
      */
-    private static void loadSystemOntology() {
-        try (GraknGraph graph = factory().open(GraknTxType.WRITE)) {
+    private static void loadSystemOntology(InternalFactory factory) {
```

Comment:
Then the order of loading is incorrect. The factory should always be initialized before attempting to load the ontology.